spec/support/embeds_helpers.rb

```diff
@@ -0,0 +1,11 @@
+module EmbedsHelpers
+  def stub_request_head(url, status_code = 200)
+    stub_request(:head, url)
+      .with(
+        headers: {
+          Accept: "*/*",
+          "User-Agent": "Ruby"
+        },
+      ).to_return(status: status_code, body: "", headers: {})
+  end
```

Comment:
I like that you kept this simple instead of trying to anticipate all possible ways this could be customized in the future. I probably would've fallen into that trap. 😄